lisk-dex-electron/src/Orderbook.js

```diff
@@ -9,8 +10,10 @@ export default class Orderbook extends React.Component {
   }
 
   render() {
+    
     if (this.props.side === 'asks') {
-      const orders = this.props.orderBookData.asks.map(order => (
+      const groupedAskOrders = Values(groupByKey(this.props.orderBookData.asks, "price", "sizeRemaining"))
```

Comment:
We should use consistent style for strings. Currently, it's single quote `'` for logic and double quotes `"` only for HTML attributes.